.github/workflows/check.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
     runs-on: ${{matrix.os}}
+    timeout-minutes: 60
```

Comment:
Not all that arbitrary...checked most of the run time in the past and it falls under ~30mins. Glancing around on agave and only a handful of runs push pass 1hr mark.